### PR TITLE
[k8s.io/client-go]fix typo in eventseries_test.go method

### PR DESCRIPTION
**Pull request by tariq1890**
/kind cleanup

```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/client-go/tools/events/eventseries_test.go [hunk 1]
```diff
@@ -168,17 +168,17 @@ func TestEventSeriesf(t *testing.T) {
 		if item.expectUpdate {
 			actualEvent := <-patchEvent
 			t.Logf("%v - validating event affected by patch request", index)
-			validateEventSerie(strconv.Itoa(index), true, actualEvent, item.expect, t)
+			validateEventSeries(strconv.Itoa(index), true, actualEvent, item.expect, t)
```
Comment (yastij) [inline staging/src/k8s.io/client-go/tools/events/eventseries_test.go]: actually, we should just rename this to `validateEvent`

**Review by yastij — CHANGES_REQUESTED** [review]

File: staging/src/k8s.io/client-go/tools/events/eventseries_test.go (hunk 1 above)
Comment (tariq1890) [reply staging/src/k8s.io/client-go/tools/events/eventseries_test.go]: done

**Review by yastij — COMMENTED** [review]
/lgtm
/approve
/priority backlog

thanks !

**Pull request merged**